src/libcollections/string.rs

```diff
@@ -1559,6 +1559,14 @@ impl<'a> Add<&'a str> for String {
     }
 }
 
+#[stable(feature = "rust1", since = "1.11.0")]
```

Comment:
Ah it's ok to pick an arbitrary feature name here, and the since I believe at this point should be 1.12.0